### PR TITLE
adding geopandas for dual travis testing

### PR DESCRIPTION
**Pull request by jGaboardi**
adding `geopandas` to `.travis.yml` as per [this](https://github.com/ljwolf/submodule_template/commit/4fcd22f65652ece3a33e0a3d09dd3540d33b7f55).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +36,7 @@ before_install:
 
 install:
   - conda install --yes pip
+  - conda install --yes -c conda-forge geopandas
```
Comment (weikang9009) [inline .travis.yml]: installing `geopandas` for travis testing is unnecessary if `spaghetti` does not rely on `geopandas` as a dependency.

Comment (ljwolf) [reply .travis.yml]: I agree... That test wouldn't run if you did not install geopandas.

Comment (weikang9009) [reply .travis.yml]: Sounds great! Sorry that I did not realize `spaghetti` is relying on `geopandas`.

Comment (jGaboardi) [reply .travis.yml]: @weikang9009 no problem at all. Please always give me feedback if you think something is amiss.

**Pull request merged**